### PR TITLE
Feature/style autosuggest input

### PR DESCRIPTION
**Pull request by LittleSweetGun**
goodhive.job_offers table schema for now:

CREATE TABLE goodhive.job_offers (
    id SERIAL PRIMARY KEY,
    title VARCHAR(100) NOT NULL,
    type_engagement VARCHAR(20) NOT NULL,
    description VARCHAR(255),
    duration VARCHAR(100) NOT NULL,
    rate_per_hour INT,
    budget VARCHAR(100),
    skills VARCHAR(255)
);

Will have to add a list of suggestions for type_engagement if we want it as an ENUM type.

### CODE REVIEW

**Comment by LittleSweetGun** [comment]
#73

File: app/companies/create-job/page.tsx [hunk 1]
```diff
@@ -52,13 +57,13 @@ export default function CreateJob() {
 
   //TODO: Put the following code in a Autosuggest Input component
   const languages = [
```
Comment (n1c01a5) [inline app/companies/create-job/page.tsx:59]: Move this to global vars.

`NEXT_PUBLIC_SKILLS=Javascript,Solidity,SQL`

Rename to `skills`.

File: app/companies/create-job/page.tsx [hunk 2]
```diff
@@ -92,14 +97,17 @@ export default function CreateJob() {
       event: React.FormEvent<HTMLInputElement>,
       { suggestion }: Autosuggest.SuggestionSelectedEventData<any>
     ) => {
-      setSelectedSkills([...selectedSkills, suggestion]);
+      if (!selectedSkills.includes(suggestion))
+        setSelectedSkills([...selectedSkills, suggestion]);
       setValue("");
```
Comment (n1c01a5) [inline app/companies/create-job/page.tsx:102]:
```
if (selectedSkills.includes(suggestion)) return setValue("");

setSelectedSkills([...selectedSkills, suggestion]);
```

File: app/companies/create-job/page.tsx [hunk 3]
```diff
@@ -111,19 +119,21 @@ export default function CreateJob() {
         setValue(newValue);
       },
       className:
-        "border-b border-gray-300 block w-full px-4 py-2 text-base font-normal text-gray-600 bg-clip-padding hover:shadow-lg transition ease-in-out m-0 focus:text-black focus:bg-white focus:outline-none focus:ring-0",
+        " rounded-full block me-5 w-full px-4 py-2 text-base font-normal text-gray-600 bg-clip-padding transition ease-in-out focus:text-black bg-gray-100 focus:outline-none focus:ring-0",
```
Comment (n1c01a5) [inline app/companies/create-job/page.tsx:122]: Remove useless ` `.

**Review by n1c01a5 — CHANGES_REQUESTED** [review]

**Comment by LittleSweetGun** [comment]
Add global var `skills` in .env:
NEXT_PUBLIC_SKILLS=JavaScript,Solidity,SQL,Tailwind,EtherJS,TypeScript,IPFS,ReactJS,NextJS

**Review by n1c01a5 — CHANGES_REQUESTED** [review]

**Comment by n1c01a5** [comment]
UX error:

- [ ] proposals of the select have the save background on the input so we cannot distinct both sections
- [ ] when we write something and enter it's remove the last item

@LittleSweetGun can you fix these bugs ?

File: app/companies/create-job/page.tsx [hunk 4]
```diff
@@ -1,12 +1,14 @@
 "use client";
 
-import { useState, FormEvent } from "react";
+import { useRef, useState, FormEvent, ChangeEvent } from "react";
```
Comment (n1c01a5) [inline app/companies/create-job/page.tsx:3]: `useRef` is useless.

**Review by n1c01a5 — CHANGES_REQUESTED** [review]

**Review by n1c01a5 — CHANGES_REQUESTED** [review]

File: app/companies/create-job/page.tsx [hunk 5]
```diff
@@ -51,19 +56,7 @@ export default function CreateJob() {
...
-    "C#",
-    "Tailwind",
-    "EtherJS",
-    "TS",
-    "IPFS",
-    "ReactJS",
-    "NextJS",
-    // Add more skills here if needed...
-  ];
+  const skills = process.env.NEXT_PUBLIC_SKILLS?.split(",") ?? [];
```
Comment (n1c01a5) [inline app/companies/create-job/page.tsx:59]: Move this to `constants/skills.js`

**Review by n1c01a5 — CHANGES_REQUESTED** [review]

File: app/companies/create-job/page.tsx [hunk 6]
```diff
@@ -56,8 +55,6 @@ export default function CreateJob() {
   };
 
   //TODO: Put the following code in a Autosuggest Input component
-  const skills = process.env.NEXT_PUBLIC_SKILLS?.split(",") ?? [];
-
   const AutosuggestInput = () => {
```
Comment (n1c01a5) [inline app/companies/create-job/page.tsx:58]: Must `camelCase` name format, AutosuggestInput => autosuggestInput .

**Review by n1c01a5 — CHANGES_REQUESTED** [review]

**Pull request merged**